script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -272,6 +272,7 @@ Obviously, I also sprinkle in the adjective 'super cool'.
 -- Page 115, Offset 22433.
 -- ここから杭を打ち出して嘘つき……いえ、吸血鬼におしおきをする道具ですね。油圧式でしたが、それでは特殊環境化で故障する危険性が高いのでメイン動力は魔力による外部圧力に作り直しています。今まで三体もの大物を仕留めた、わたしの自慢の一品なんですよ」
 // 三体もの大物 being Be'ze/Einnashe/Chromclay
+// I see that "external pressure" is the correct translation, but wouldn't something like this be powered by internal pressure like a gun?
```

Comment:
I assumed it meant that the magical energy was being supplied from outside (from her)